perf/benchmark/run_benchmark_job.sh

```diff
@@ -117,12 +117,12 @@ function setup_fortio_and_prometheus() {
 
 #TODO: add stackdriver filter
 function prerun_v2_nullvm() {
-  export SET_OVERLAY="values.telemetry.enabled=true,values.telemetry.v1.enabled=false,values.telemetry.v2.enabled=true,values.telemetry.v2.prometheus.enabled=true"
+  export SET_OVERLAY=""
```

Comment:
What about 1.4? Will it break that ?

Reply:
I think it is ok to leave it in.
We are testing a specific configuration here, not necessarily default.

Reply:
it will not backport to 1.4, but it is true that it is not necessarily default. 

But I will change the set format because of this issue: https://github.com/istio/istio/issues/20916